packages/video_player/video_player/pubspec.yaml

```diff
@@ -25,11 +25,13 @@ dependencies:
   flutter:
     sdk: flutter
   html: ^0.15.0
+  collection: ^1.19.0
```

Comment:
You need to run the analyze tool as well to see what's failing - currently it's this line. These deps should be in alphabetical order.

Reply:
https://github.com/flutter/plugins/blob/master_archive/script/tool/README.md#commands

Reply:
Thank you for the link, I sorted them.